### ISSUE TITLE
Homebrew package no longer maintained

### ISSUE THREAD

**Issue by asdfgeoff** [xtin/TeleFred#6]
Running `brew install telegram-cli` yields the following error:

```
Error: telegram-cli has been disabled because it is not maintained upstream!
```

Should this repo be marked as unmaintained, to match?

**Comment by xtin** [comment]
Yeah, I'm not using Alfred anymore, hence I won't update it in the future. But as you can read in 0.0.4 I tried bundling it with another telegram-cli
